.gitignore

```diff
@@ -27,4 +27,5 @@ htmlcov
 python-env
 
 ua-converter/user-agents.json
-ua-converter/browsers.jsonl
\ No newline at end of file
+ua-converter/browsers.jsonl
+test_script.py
```

Comment:
In other projects, I have a `.personal_workspace/` line here instead so I can add any files I want to that directory.

Reply:
Good idea